ietf/templates/doc/document_info.html

```diff
@@ -79,14 +79,13 @@
                 {% include "doc/revisions_list.html" with document_html=document_html %}
             </td>
         </tr>
-        {% if doc.rev != "00" %}
+        {% if doc.rev != "00" and diff_revisions|length > 1 %}
```

Comment:
This prevents me from diffing a new wg-00 with whatever nonwg draft it replaces.